homeassistant/helpers/selector.py

```diff
@@ -179,8 +179,8 @@ class NumberSelector(Selector):
 
     CONFIG_SCHEMA = vol.Schema(
         {
-            vol.Required("min"): vol.Coerce(float),
-            vol.Required("max"): vol.Coerce(float),
+            vol.Optional("min"): vol.Coerce(float),
+            vol.Optional("max"): vol.Coerce(float),
```

Comment:
Frontend marks these as optional

Reply:
Hmmm, I think that is not right.
This is especially problematic with sliders I would say...

I think, in this case, the frontend is incorrectly typed?